config-EXAMPLE.json

```diff
@@ -28,10 +28,10 @@
     "hypixelAPIkey": "HYPIXEL_API_KEY",
 
     "skyHelperAPI": "http://192.95.42.73:1924//v1",
-    "skyHelperKey": "DuckySoLucky",
+    "skyHelperKey": "DuckySoLucky", // leave this
```

Comment:
Comments don't exist in JSON files, so this is gonna cause an error. Also can you please rename file to `config.example.json`